ob1k-consul/src/main/java/com/outbrain/ob1k/consul/ServiceRegistration.java

```diff
@@ -18,7 +18,11 @@
 
   // I need to make some marshalers happy ;)
   ServiceRegistration() {
-    this(null, 0, null, null, null);
+    name = null;
```

Comment:
No Michael. 
If your approach is that you know better and findbugs is the annoying thing warning about meaningless stuff then there's no point in running it. We might as well remove it. 
Its a tool that is there to help you. So "meh" is wrong here.

findbugs is dead right here and points to a real problem in the code. 

There are two options here:
1. If marshalling is not needed then remove it.
2. If it is needed then it shouldn't throw an NPE. 

I think this fix points at option 2 but I would not use null when there's another constructor that checks that the field should never be null. I would use an empty string but I guess that is more style than correctness.